dino.icu.yaml

```
@@ -398,6 +398,11 @@ scales: # (yet another?) currency slack bot (with some web UI) :D
   type: A
   value: 150.136.254.91 # Oracle Always Free:tm: server lol
 
+subdomain: # Free subdomain generator.
+  ttl: 600
+  type: A
+  value: 76.76.21.21
```

I assume this is Vercel. If so, you should use a CNAME instead of an A record
```suggestion
  type: CNAME
  value: cname.vercel-dns.com.
```